src/app/collection/pages/security-injections/security-injections.component.ts

```diff
@@ -1,15 +1,43 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, OnDestroy, HostListener } from '@angular/core';
+import { Router } from '@angular/router';
+import { LearningObject } from '@entity';
+import { CollectionService } from 'app/core/collection.service';
+import { NavbarService } from '../../../core/navbar.service';
 
 @Component({
   selector: 'clark-security-injections',
   templateUrl: './security-injections.component.html',
-  styleUrls: ['./security-injections.component.scss']
+  styleUrls: ['./security-injections.component.scss'],
 })
-export class SecurityInjectionsComponent implements OnInit {
+export class SecurityInjectionsComponent implements OnInit, OnDestroy {
 
-  constructor() { }
+  abvCollection = 'secinj';
+  collection;
+  learningObjects: LearningObject[];
+  isDesktop: boolean;
+
+  constructor(
+    private navbarService: NavbarService,
+    private collectionService: CollectionService,
+    private router: Router,
+    ) {
+      this.isDesktop = window.innerWidth >= 768;
+    }
+
+    @HostListener('window:resize', ['$event'])
+    onResize(event: Event) {
+    this.isDesktop = window.innerWidth >= 768;
+    }
+
+  async ngOnInit() {
+
+    this.navbarService.show();
+
+    this.collection = await this.collectionService.getCollection(this.abvCollection);
```

Comment:
I would check if the description is retrieved with `this.collectionService.getCollection()` on the security injections page. I think `this.collectionService.getCollectionMetadata()` would get the metadata which includes the description.
```suggestion
    this.collection = await this.collectionService.getCollectionMetadata(this.abvCollection);
```